kie-wb-parent/kie-wb-webapp/src/main/resources/org/kie/workbench/KIEWebapp.gwt.xml

```diff
@@ -99,6 +99,8 @@
   <inherits name="org.drools.workbench.screens.dtablexls.DroolsWorkbenchDecisionTableXLSEditorClient"/>
   <inherits name="org.drools.workbench.screens.scorecardxls.DroolsWorkbenchScoreCardXLSEditorClient"/>
   <inherits name="org.drools.workbench.screens.testscenario.DroolsWorkbenchTestScenarioEditorClient"/>
+  <inherits name="org.drools.workbench.screens.scenariosimulation.DroolsWorkbenchScenarioSimulationEditorAPI" />
```

Comment:
You probably don't need to inherit API as Client (probably) inherits already.

Reply:
True